test/test_limited_api.py

```diff
@@ -0,0 +1,41 @@
+import textwrap
+
+from . import test_projects, utils
+
+limited_api_project = test_projects.new_c_project(
+    setup_cfg_add=textwrap.dedent(r'''
+        [bdist_wheel]
+        py_limited_api=cp36
+    ''')
+)
+
+
+def test_setup_cfg(tmp_path):
+    project_dir = tmp_path / 'project'
+    limited_api_project.generate(project_dir)
+
+    # build the wheels
+    actual_wheels = utils.cibuildwheel_run(project_dir, add_env={
+        'CIBW_BUILD': 'cp27-* cp36-*',  # PyPy does not have a Py_LIMITED_API equivalent
```

Comment:
Double checking: @mattip, I'm correct that PyPy doesn't have something like `Py_LIMITED_API`? The only reference I could find in the PyPy repository seemed to be cffi-related, but I didn't do a full in-depth search.

Reply:
Right, PyPy doesn't do Py_LIMITED_API yet. @mattip and I were chatting about this the other day:

https://github.com/pallets/markupsafe/issues/175#issuecomment-770254962

Reply:
Thanks!